modules/manage/pages/manage-nodes/add-node-and-rebalance.adoc

```diff
@@ -24,7 +24,7 @@ The examples on this page assume that the default, _system-generated_ certificat
 In a production or similar context, to ensure security, only administrator-configured certificates should be used on a cluster: these should rely on a well known _Certificate Authority_, whose certificate is loaded as the cluster's _root_ certificate.
 (For more information, see xref:learn:security/certificates.adoc#server-certificates[Default Certificates and Certificate Substitution].)
 
-In such a context, no node can be added to the cluster until conformant administrator-configured certificates have been loaded onto it -- such activities will thus need to be performed in addition to those shown by the examples on this page.
+In such a context, no node can be added to the cluster until conformant administrator-configured certificates have been loaded onto it such activities must be performed in addition to those shown by the examples on this page.
```

Comment:
It seems like there's a semicolon missing after "loaded onto it":

In such a context, no node can be added to the cluster until conformant administrator-configured certificates have been loaded onto it; such activities must be performed in addition to those shown by the examples on this page.